herd/AArch64Sem.ml

```diff
@@ -45,6 +45,10 @@ module Make
     let const_pac_field = C.variant Variant.ConstPacField
     let fpac = C.variant Variant.FPac
 
+    let check_kvm ins =
```

Comment:
I wonder why not to do this check at the beginning of `lift_kvm`. The instruction is known via `ii` there.

Reply:
I think that all cases except for AT will call lift_kvm only if kvm is true so the check wouldn't add much.